teamcity: failed test: TestDockerCLI

The following tests appear to have failed on master (acceptance): TestDockerCLI, TestDockerCLI/test_demo_workload.tcl

You may want to check [for open issues](https://github.com/cockroachdb/cockroach/issues?q=is%3Aissue+is%3Aopen+TestDockerCLI).

[#1507576](https://teamcity.cockroachdb.com/viewLog.html?buildId=1507576):


```
TestDockerCLI/test_demo_workload.tcl
--- FAIL: acceptance/TestDockerCLI: TestDockerCLI/test_demo_workload.tcl (8.380s)




TestDockerCLI
--- FAIL: acceptance/TestDockerCLI (324.350s)

------- Stdout: -------
    test_log_scope.go:77: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/logTestDockerCLI093138227
    test_log_scope.go:58: use -show-logs to present logs inline




```

Please assign, take a look and update the issue accordingly.

--- a/pkg/cli/interactive_tests/test_demo_workload.tcl
+++ b/pkg/cli/interactive_tests/test_demo_workload.tcl
@@ -4,6 +4,12 @@ source [file join [file dirname $argv0] common.tcl]
 
 start_test "Check cockroach demo --with-load runs the movr workload"
 
+# Disable trying to acquire the demo license. This test does not
+# need enterprise features, and sometimes if the licensing server
+# is unavailable, the error message from failing to receive the
+# testing license pollutes the test output.
+set env(COCKROACH_SKIP_ENABLING_DIAGNOSTIC_REPORTING) "true"
+
 # Start demo with movr and the movr workload.
 spawn $argv demo movr --with-load
 

